processor/src/main/java/org/mapstruct/ap/internal/model/annotation/BooleanProperty.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.internal.model.annotation;
+
+import java.util.List;
+
+/**
+ * @author Ben Zegveld
+ */
+public class BooleanProperty extends Property {
```

Comment:
keep the names consistent here with here with the java language. Like `BooleanAnnotationElement extends AnnotationElement`